src/pretix/base/models/items.py

```diff
@@ -476,6 +498,19 @@ def __lt__(self, other):
             return self.id < other.id
         return self.position < other.position
 
+    @staticmethod
+    def clean_order_positions(variation):
+        if not len(variation.orderposition_set.all()):
```

Comment:
``if not variation.orderposition_set.exists()`` is more efficient **here**. It might be less efficient in other places since ``.all()`` might already be cached due to ``prefetch_related`` calls, but that doesn't affect us here.